ntbs-service/Models/SocialContextVenue.cs

```diff
@@ -70,9 +70,15 @@ public class SocialContextVenue : ModelBase, IHasPostcode
         [NotMapped]
         public DateTime? Dob { get; set; } = null;
 
-        [NotMapped]
         public bool DateFromAfterDob => Dob == null || DateFrom >= Dob;
-        [NotMapped]
         public bool DateToAfterDob => Dob == null || DateTo >= Dob;
+
+        public string FormattedDateFrom => FormatDate(DateFrom);
+        public string FormattedDateTo => FormatDate(DateTo);
+
+        private string FormatDate(DateTime? dateTime)
+        {
+            return dateTime?.ToString("dd MMM yyyy");
+        }

```

Comment:
This isn't actually `SocialContextVenue` specific - do we want a general helper for this?